attributes/default.rb

```diff
@@ -120,12 +120,19 @@
 default['mysql']['scripts']          = %w{ get-mysql-socket.sh get-mysql-port.sh mysql-server-start.sh mysql-server-stop.sh mysql-server-restart.sh mysql-client.sh }
 
 default['ndb']['dir']                                 = node['install']['dir'].empty? ? "/var/lib" : node['install']['dir']
+
 default['ndb']['root_dir']                            = "#{node['ndb']['dir']}/mysql-cluster"
 default['ndb']['log_dir']                             = "#{node['ndb']['root_dir']}/log"
 default['ndb']['data_dir']                            = "#{node['ndb']['root_dir']}/ndb_data"
 default['ndb']['version_dir']                         = "#{node['ndb']['root_dir']}/ndb-#{node['ndb']['version']}"
 default['ndb']['base_dir']                            = "#{node['ndb']['root_dir']}/ndb"
 
+# Data volume directories
+default['ndb']['data_volume']['root_dir']             = "#{node['data']['dir']}/rondb"
```

Comment:
What about the `mysql` data directory in `/srv/hops/mysql-cluster/mysql` - that should also be in the data dir. Otherwise you'll need to restore users and view every time you patch hopsworks.ai

Reply:
At this point I'm wondering, why don't we just move the entire `mysql-cluster` dir?

Reply:
Correct, `mysql` should also be moved to the data volume. The problem with moving the entire `mysql-cluster` directory is that it contains RonDB binaries. If we move it outside of `mysql-cluster` I think it would work.